hardware-and-software-requirements.md

```diff
@@ -66,7 +66,8 @@ In v8.1 LTS, TiDB ensures multi-level quality standards for various combinations
 
     > **Note:**
     >
-    > According to [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/), the upstream support for CentOS Linux 8 ended on December 31, 2021. CentOS Stream 8 continues to be supported by the CentOS organization.
+    > - According to [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/), the upstream support for CentOS Linux 8 ended on December 31, 2021. CentOS Stream 8 continues to be supported by the CentOS organization.
```

Comment:
This isn't true anymore: CentOS Stream 8 is no longer supported by the CentOS organization.

| OS | EOL Date|
|---|---|
| CentOS 7 | June 30, 2024 |
| RHEL 7 | June 30, 2024 (offically this is EOM, but that's mostly the same) |
| CentOS Linux 8 | December 31, 2021 |
| CentOS Stream 8 | May 31, 2024 |

- https://www.centos.org/centos-linux-eol/
- https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/
- https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux/rhel-7-end-of-maintenance
- https://endoflife.date/rhel
- https://endoflife.date/centos
- https://endoflife.date/centos-stream
- https://endoflife.date/rocky-linux

Reply:
Got it. I'll update it.